src/state_transition/block/process_attestation_phase0.zig

```diff
@@ -3,6 +3,7 @@ const Allocator = std.mem.Allocator;
 const CachedBeaconStateAllForks = @import("../cache/state_cache.zig").CachedBeaconStateAllForks;
 const BeaconStateAllForks = @import("../types/beacon_state.zig").BeaconStateAllForks;
 const ssz = @import("consensus_types");
+const s = @import("ssz");
```

Comment:
I think in near future we should consider what we want to do with namespaces, right now we're using `ssz` for consensus types but then we run into situations like this where we can't use `ssz` the library.

Should we just use `ct` for `consensus_types` and `ssz` to mean `ssz` the library?

Reply:
Prefer to reserve `ssz` for ssz library, and something else for consensus_types

Reply:
tracked in https://github.com/ChainSafe/lodestar-z/issues/82